easybuild/easyconfigs/n/NextPolish/NextPolish-1.4.1-GCC-11.2.0.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'Bundle'
+
+name = 'NextPolish'
+version = '1.4.1'
+
+homepage = "https://github.com/Nextomics/NextPolish"
+description = """NextPolish is used to fix base errors (SNV/Indel) in the genome generated by noisy long reads, it can
+be used with short read data only or long read data only or a combination of both. It contains two core modules, and use
+a stepwise fashion to correct the error bases in reference genome."""
```

Comment:
@seb45tian I'm sure you've copied this from upstream, but it reads a bit oddly at the end.
Should it be something like "and they use a stepwise fashion to correct the error bases in the reference genome"?

Reply:
Ah yes, its directly copied from upstream and sounds a bit *off*. But I would suggest

> It contains two core modules, and **is used** in a stepwise fashion to correct the error bases in **the** reference genome.

This makes the most sense to me, however I will say that I am definitely no expert in gnome sequencing etc. If you ok with this, I'll do a new commit.

Reply:
Yes, that reads much better.